README.md

```diff
@@ -23,7 +23,23 @@ Sample Scanner Implementation
 
 Development Requirements
 -------------
-On fedora you can install requirements by issuing
+On Fedora or RHEL you can install requirements by issuing
 
-    dnf install ruby-devel rubygem-ffi rubygem-rake rubygem-bundler openscap
+    yum install ruby-devel rubygem-ffi rubygem-rake rubygem-bundler openscap
+
+Test Requirements
+-------------
+For tests on RHEL7, you need minitest package and specific older version of rubocop.
+
+    yum install rubygem-minitest
+    gem install rubocop -v 0.42.0
```

Comment:
I wonder, why did you decide to lock on certain version?

Can ruby-openscap upstream rely on you to maintain this dependency until eternity? :-)

Reply:
@isimluk there is old version of ruby in RHEL7 with which upstream rubocop does not work. Thus I have selected rubocop version that works there. Not sure what other solution we have.

Reply:
Found out which rubocop is newest available, set the version, and explained a bit more why this problem happens :)